dlog/tests/batch.rs

```diff
@@ -31,8 +31,14 @@ where
     let rng = &mut rand::thread_rng();
     let mut random = rand::thread_rng();
 
-    let size = 1 << 7;
+    let size_log2 = 7;
+    let padding_to_test_trimming = 2;
+    let size = 1 << (size_log2 + padding_to_test_trimming);
+
+    // We make the srs larger than the "trimmed" size in order to
+    // test the trimming functionality.
     let srs = SRS::<Affine>::create(size);
+    let srs = srs.trim(size_log2);
```

Comment:
at this point wouldn't it be better if `SRS::create` returned a trimmed SRS?

Reply:
The goal with the trimming is to have one big global SRS (say 2^17 or something) and then trim it down for smaller circuits so that they all share the memory.

Reply:
Oh! I see, that makes more sense.